java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md

```diff
@@ -78,4 +78,37 @@ bin/start.sh \
 --templateProperty gcs.jdbc.output.url='jdbc:mysql://192.168.16.3:3306/test?user=root&password=root' \
 --templateProperty gcs.jdbc.output.driver='com.mysql.jdbc.Driver'
 
+```
+
+## 4. GCS to GCS
+
+```
+
+GCP_PROJECT=<gcp-project-id> \
+REGION=<region>  \
+GCS_STAGING_LOCATION=<gcs-staging-bucket-folder> \
+bin/start.sh \
+-- --template GCSTOGCS \
+--templateProperty project.id=<gcp-project-id> \
+--templateProperty gcs.gcs.input.format=<avro | parquet | orc> \
+--templateProperty gcs.gcs.input.location=<gcs path> \
+--templateProperty gcs.gcs.output.format=<avro | parquet | orc> \
+--templateProperty gcs.gcs.output.location=<gcs path> \
+--templateProperty gcs.gcs.write.mode=<Append|Overwrite|ErrorIfExists|Ignore>
+--templateProperty gcs.gcs.temp.table=<temporary table name for data processing> \
+--templateProperty gcs.gcs.temp.query=<sql query to process data from temporary table>
+
+Example execution:-
+
+bin/start.sh \
+-- --template GCSTOJDBC \
+--templateProperty project.id=my-gcp-project \
+--templateProperty gcs.gcs.input.location=gs://my-gcp-project-input-bucket/empavro \
```

Comment:
don't you needto pass absolute filename like:
`--templateProperty gcs.gcs.input.location=gs://my-gcp-project-input-bucket/empavro.avro`
or
`--templateProperty gcs.gcs.input.location=gs://my-gcp-project-input-bucket/empavro*.avro`

Reply:
changed specific file name to a generic one.